libs/src/services/creatorNode/CreatorNodeSelection.js

```diff
@@ -216,6 +217,32 @@ class CreatorNodeSelection extends ServiceSelection {
 
     this.decisionTree.push({ stage: DECISION_TREE_STATE.FILTER_OUT_UNHEALTHY_OUTDATED_AND_NO_STORAGE_SPACE, val: healthyServicesList })
 
+    // Record metrics
+    if (this.creatorNode.monitoringCallbacks.healthCheck) {
```

Comment:
@raymondjacobson what are these callbacks? are they calling external services / do they need to be try/catched?

Reply:
okay i found out what they are https://github.com/AudiusProject/audius-protocol/pull/1207/files#diff-b1e49d4c865a8d40686e05f6ac646d90ecdbd5e882f6179f5ceffbc5a549af04R88-R90. just curious if they need try/catching

Reply:
they're methods that should be O(1) (not async/awaiting calling outbound analytics services) so they really shouldn't throw, but i can add try catches just to be extra safe